account_invoice_cancel_reason/view/account_invoice_view.xml

```diff
@@ -32,7 +32,9 @@
                     <attribute name="invisible">True</attribute>
                 </xpath>
                 <xpath expr="//button[@name='invoice_cancel']" position="before">
-                    <button name="%(action_account_invoice_cancel)d" type="action" states="draft,proforma2,open" string="Cancel Invoice" groups="account.group_account_invoice"/>
+                    <!-- <button name="%(action_account_invoice_cancel)d" type="action" states="draft,proforma2,open" string="Cancel Invoice" groups="account.group_account_invoice"/> -->
+                    <button name="invoice_open" states="draft" string="Cancel Invoice"  class="oe_highlight"/>
```

Comment:
If you edit a code like this, you think you can cancel the invoices.

Reply:
Cannot cancel the invoice.

Reply:
Please edit this code :v: